tests/docker-compose.yml

```diff
@@ -17,7 +17,6 @@ services:
       retries: 3
       start_period: 30s
 
-  # TODO: remove this once voucher is deployed on bellecour

```

Comment:
a new container replacing this one should be available soon for the voucher subgraph deployment